configs/Phi-3-mini-4k-instruct/README.md

```diff
@@ -46,4 +46,47 @@ python console_chat.py
+### Prerequisites
+1. To run the model fine-tuning in your remote Azure Container App Environment, you need to make sure your subscription have enough GPU capacity amount. Submit a [support ticket](https://azure.microsoft.com/support/create-ticket/) to request the capacity amount required for your application. [Learn More about GPU capacity](https://learn.microsoft.com/en-us/azure/container-apps/workload-profiles-overview)
+2. If you are using private dataset on HuggingFace, make sure you have a [HuggingFace account](https://huggingface.co/) and [generate an access token](https://huggingface.co/docs/hub/security-tokens)
+3. If you enable [Weights & Biases](https://wandb.ai/site), make sure you have a Weights & Biases account and [retrieve your API Key](https://wandb.ai/authorize)
+
+### Provision Azure Resources
+To get started, you need provision the Azure Resource for remote fine-tuning. This can be done by running the `AI Toolkit: Provision Azure Container Apps job for fine-tuning.` from command palette.
+
+You can monitor the progress of the provision via the link that is displayed in the output channel.
+
+### Add ACA Secrets
+#### Weights & Biases API Key
+If you've enabled Weights & Biases in your project, it's crucial to set your Weights & Biases API Key for seamless access. Use the `AI Toolkit: Add Azure Container Apps Job secret for fine-tuning` command to input your Weights & Biases API Key. In this process, the secret name should be [`WANDB_API_KEY`](https://docs.wandb.ai/guides/track/environment-variables#optional-environment-variables) with your API Key as the secret value.
+
+#### [Optional] HuggingFace token
+If you're using private HuggingFace dataset, you should set your HuggingFace token as an environment variable to bypass the need for manual login on the Hugging Face Hub.
+You can do this using the `AI Toolkit: Add Azure Container Apps Job secret for fine-tuning command`. With this command, you can set the secret name as [`HF_TOKEN`](https://huggingface.co/docs/huggingface_hub/package_reference/environment_variables#hftoken) and use your Hugging Face token as the secret value.
+
+### Run Fine-tuning
+To start the remote fine-tuning job, execute the `AI Toolkit: Run fine-tuning` command.
+
+The job streaming log will be displayed in the output panel if the job has started running. 
+> **Note:** The job might be queued if there are insufficient resources available. If the log fails to display when the job starts, you can wait for a while and then execute the `AI Toolkit: Show the running fine-tuning job streaming logs` command to re-connect to the streaming log.
+    
+During this process, QLoRA will be used for fine-tuning, and will create LoRA adapters for the model to use during inference.
+The results of the fine-tuning will be stored in the Azure Files.
+
+### Provision Inference Endpoint
+After the adapters are trained in the remote environment, you can use a simple Gradio application to interact with the model.  
```

Comment:
Proposed: 
After the adapters are trained in the remote environment, use a simple Gradio application to interact with the model.